packages/http/README.md

```diff
@@ -30,3 +30,55 @@ const data = await PublicApiService.postGetWhoami({
   },
 });
 ```
+
+## HTTP fetchers
+
+`@turnkey/http` provides fully typed http fetchers for interacting with the Turnkey API. You can find all available methods [here](./src/__generated__/services/coordinator/public/v1/public_api.fetcher.ts). The types of input parameters and output responses are also exported for convenience.
+
+The OpenAPI spec that generates all fetchers is also [included](./src/__generated__/services/coordinator/public/v1/public_api.swagger.json) in the package.
+
+## `withAsyncPolling(...)` helper
+
+All Turnkey mutation endpoints are asynchronous (with the exception of signing endpoints) because we support consensus via policy on all write actions. To help you simplify async mutations, `@turnkey/http` provides a `withAsyncPolling(...)` wrapper. Here's a quick example:
```

Comment:
`because we support consensus via policy on all write actions` is misleading.

We support consensus on signing actions yet this activity type is synchronous. Why? Organization data writes! We have a queue for writes, which forces us to be asynchronous (our updater picks up tasks from the queue, decoupled from end-user requests).

Signatures don't require data changes so they can be synchronous. Whereas all other actions require a change in organization data (create user, delete policy, etc etc)

Reply:
I'd say something along the lines of: `All Turnkey mutation endpoints are asynchronous because we process updates to organization data asynchronously, with a queuing system. Signing activities are a notable exception: they do not require organization data updates, hence they are synchronous.`

Reply:
> I'm gonna further simplify this because it's mostly implementation details (in our current system)

any more context on this? by current system, do you mean the current state of the SDK, or the turnkey backend?